Prepare documentation for translation (#862)
ManimCommunity/manim: docs/source/contributing/translation.rst

You have access to the projects you joined just on the left of your profile picture
on the top bar.

.. image:: /_static/select_transifex_repo.png
   :align: center
   :width: 200

Now go to the
`languages list <https://www.transifex.com/manim-community/manim-community-documentation/languages/>`_
and select a language you want to help with. You should see a blue button reading
`Translate`; click it to get started! You will see a list of
resources in the project. You can click on any untranslated resource, then
click on `Translate`. You will then see something like the following:

.. image:: /_static/transifex_explanation.png
    :align: center

The `A` panel is the list of all the strings in the resource. Click on any of
them to translate it. It will appear in the `B` panel, and you will be able to
work on your translation in the `C` panel. Once you're done, click `Save changes`.
Your translation will have to be reviewed before being added to the docs.
You can also get back to the resources list by clicking `d` or change the
destination language in `e`. Your

If you want to become a reviewer, please join `Manim's Discord server <https://www.manim.community/discord/>`_

> and you can ping the role @i18nService.

Reviewer: I feel like a name like "Translation Admin" would be better

Author: Instead of i18nService ? You're right. How should i write the spaces ? @TranslationAdmin or @Translation Admin

Reviewer: The latter

====================
Internationalization
====================

If you want to help with translating the docs:

> #. Make sure you have a `Transifex <https://www.transifex.com/>`_ account. Transifex is a tool for colaborative translation and allows Manim team to review and help to improve suggestions from the community. You will use Transifex to submit your translation suggestions.

Reviewer: ```suggestion
#. Make sure you have a `Transifex <https://www.transifex.com/>`_ account. Transifex is a tool for colaborative translation and allows the Manim team to review translations.
```

> #. Open to the `Manim project page <https://www.transifex.com/manim-community/manim-community-documentation>`_ .

Reviewer: ```suggestion
#. Open the `Manim Transifex project page <https://www.transifex.com/manim-community/manim-community-documentation>`_ .
```

> #. Click on Join Team.

Reviewer: ```suggestion
#. Click on ``Join Team``.
```

> #. Select the language you want to help with.

Reviewer: ```suggestion
#. Select the language(s) you want to help with.
```

Author: Well, you can only select one language in the combobox, so i think language without (s) is more accurate.

Reviewer: I think "a" instead of "the" would be better in that case then.

> Congratulations ! You are now a Translator for the Manim Community project!

Reviewer: ```suggestion
Congratulations! You are now a Translator for the Manim project!
```

You have access to the projects you joined just on the left of your profile picture
on the top bar.

.. image:: /_static/select_transifex_repo.png
   :align: center
   :width: 200

> Then you can go the

Reviewer: ```suggestion
 Now go to the
```

`languages list <https://www.transifex.com/manim-community/manim-community-documentation/languages/>`_

> and select the language you want to help with. Then you have a blue button reading

Reviewer: ```suggestion
and select the language(s) you want to help with. You should see a blue button reading
```

> `Translate`, click it to get started with translating ! You will get to a list of

Reviewer: ```suggestion
`Translate`; click it to get started! You will see a list of
```

> all resources of the project. You can click on any untranslated resource, then

Reviewer: ```suggestion
resources in the project. You can click on any untranslated resource, then
```

> click on `Translate` and you will get the following page:

Reviewer: ```suggestion
click on `Translate`. You will then see something like the following:
```

.. image:: /_static/transifex_explanation.png
    :align: center

The `A` panel is the list of all the strings in the resource. Click on any of
them to translate it. It will appear in the `B` panel, and you will be able to

> set your translation in the `C` panel. Once you're done, click `Save changes`.

Reviewer: ```suggestion
work on your translation in the `C` panel. Once you're done, click `Save changes`.
```

> Your translation will have to be reviewed before getting in the docs.

Reviewer: ```suggestion
Your translation will have to be reviewed before being added to the docs.
```

You can also get back to the resources list by clicking `d` or change the
destination language in `e`. Your

> If you want to become a reviewer, please join the discord server of Manim and

Reviewer: Can you add a link like how it's done in other doc pages?

> you can ping the role @i18nService.

Reviewer: Not sure if we need a specific role for this? I think we can just have moderators handle it.

Author: Well, it is about Transifex moderators that have to manually change the role of the translator to reviewer. Moderators on discord might not be moderators on Transifex.

Reviewer: Fair enough, but I think a name like "Translation admin" would be better.

If you want to add a new language, you can go to
`the project page <https://www.transifex.com/manim-community/manim-community-documentation>`_

> If you didn't joined the project yet, you will have a link which reads

Reviewer: ```suggestion
If you haven't joined the project yet, you will have a link which reads
```

> `Request language`, and send the request.

Reviewer: ```suggestion
`Request language` which you can click.
```

> If you are already in the project, go to the languages page and you have a

Reviewer: ```suggestion
If you are already in the project, this button will be in the languages page.
```

> blue button reading `Request language` and do your request.

Reviewer: ```suggestion
```

Reviewer 2: That edit changes it to:
If you are already in the project, go to the languages page and you have a
...
...

Reviewer: No, combined with the suggestion on the previous line it should be correct.

Reviewer 2: Ohh I see. To be honest, those suggestions should have been combined.

Reviewer: Ah, I didn't know we could do multiple line suggestions, but I do now. Thanks!

> We will have to manually accept the new language, so you will not be able to

Reviewer: I feel like this is obvious and does not need to be stated.

Author: I'd rather have contributors warned that they cannot translate immediately after requesting a new language.

Author: How should this be settled ?

Reviewer: Sure, I'll defer to your judgement.
